SCMS.Services.Api/Migrations/20211128141148_AddAddtionalDetailModel.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
```

Comment:
Add copyrights header.

Reply:
I replace the the migrations file with a new seeing that I needed to change the class name because of the typo